sandbox/extension/Humboldt_2023-09-01.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='/style/human.xsl'?>
+<extension xmlns='http://rs.gbif.org/extension/'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:dc='http://purl.org/dc/terms/'
+    xmlns:eco="http://rs.tdwg.org/eco/terms/"
+    xsi:schemaLocation='http://rs.gbif.org/extension/ http://rs.gbif.org/schema/extension.xsd'
+    dc:title='Humboldt Ecological Inventory'
+    name="HumboldtEcologicalInventory" namespace='http://rs.tdwg.org/eco/terms/'
+    rowType='http://rs.tdwg.org/dwc/terms/Event'
+    dc:issued='2023-09-01'
+    dc:subject='dwc:Event'
+    dc:relation='https://github.com/tdwg/hc/blob/master/vocabulary/term_versions.csv'
+    dc:description='Extended support for dwc:Events related to ecological inventories.'>
+
+    <!-- Site -->
+    <property group='Site' name='siteCount' namespace='http://rs.tdwg.org/eco/terms/' qualName='http://rs.tdwg.org/eco/terms/siteCount' dc:relation='https://tdwg.github.io/eco/terms/#eco:siteCount' dc:description='Total number of individual sites surveyed during the dwc:Event.' comments='Site refers to the location at which observations are made or samples/measurements are taken. The site can be at any level of hierarchy.' examples='`1`; `15`' required='false'/>
```

Comment:
I opened https://github.com/tdwg/hc/issues/82 so we can avoid github.io links in the extension, though it isn't especially important, and in any case this is only in the sandbox for now.